toolchain/check/cpp/type_mapping.cpp

```diff
@@ -180,6 +180,22 @@ static auto MapNonWrapperType(Context& context, SemIR::InstId inst_id,
   }
 }
 
+// If `type_id` is `Cpp.void` and the last wrapper type in `wrapper_types` is a
+// pointer, pops the pointer wrapper and returns non nullable `void*` QualType.
+// Otherwise returns a null QualType.
```

Comment:
Just to note, I would've found this comment more helpful if it'd been a little more high level. Right now it seems to be just documenting the if/else in detail. For a suggestion on a different comment style:

```suggestion
// Returns `void*` if the type is a wrapped `Cpp.void*`, consuming the
// pointer from `wrapper_types`. Otherwise returns no type.
```

Reply:
Done.